executors/dbfixtures/executor.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/mitchellh/mapstructure"
 	"github.com/ovh/venom"
 	"github.com/ovh/venom/executors"
+	migrate "github.com/rubenv/sql-migrate"
```

Comment:
"external" package import should be separated by a blank line from venom package

Reply:
Yeah nice catch. Fixed.